src/openqaoa-azure/README.md

```diff
@@ -0,0 +1,100 @@
+<div align="center">
```

Comment:
This readme probably needs an update

Reply:
Yes! All the READMEs for each plugin needs to be updated.

Reply:
resolving the conversation because it was decided that we will fix the README's in a separate PR